### DISCUSSION TITLE
トラックボール側がトラックボールなし側基盤として認識される

### DISCUSSION THREAD

**Discussion by hiro1115** [Yowkees/keyball#472] [Q&A]
keyball44を作成しています。
割とスムーズに行ったと思ったのですが、右手側(.トラックボール側)基盤がトラックボール側として読み込まれていません。トラックボールの反応もなく、何が原因かがわからず困っています。お力添えを頂けますと幸いです。

![image](https://github.com/Yowkees/keyball/assets/142844754/48f78663-7cac-48a6-9fa9-6c58a2d587df)

![image](https://github.com/Yowkees/keyball/assets/142844754/530ac7e8-1355-4f9c-b984-5bd8d2bbf6db)![image](https://github.com/Yowkees/keyball/assets/142844754/a539aff1-dc9c-453e-a3f1-bd75c591d1dd)
![image](https://github.com/Yowkees/keyball/assets/142844754/7dbc10a9-598e-41a5-9e9f-9cafb250fc4f)

→
remapの設定を変えていないだけでした。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

Keyball起動後にProMicroがトラックボール基板と通信して、反応がないと、左手基板と認識されます。
そのためトラックボールが正常になれば右手と認識されると思います。

光の加減かも知れませんが、下記画像の丸で示した部分がショートしていないかご確認お願いします。
あとは、トラックボール基板の半田付けを見直してください。トラボ基板を抜き差しするだけでも正常に戻る場合もあります。

お手数ですがご確認よろしくお願いいたします。

![KB44_trackballfix](https://github.com/Yowkees/keyball/assets/84209772/3547f384-fa5e-4c56-bb9f-dc02dd18965e)
